### PR TITLE
Revamp the Teams and Orgs page

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

- Revamp the Teams Orgs page
- Move the topic higher up in the toc for visibility
- Remove outdated screenshots
- Add new optimized screenshots

### CODE REVIEW

**Comment by MatthewHNH** [comment]
Add a link to the "Convert user account to Org" doc where it says "The organization name cannot be the same as your Docker ID" 

Add comment about registering an account before accepting an org invite

File: docker-hub/orgs.md [hunk 1]
```diff
@@ -9,151 +9,228 @@ redirect_from:
...
+
+A **Team** is a group of Docker users that belong to an organization. An
+organization can have multiple teams. When you first create an organization,
+you’ll see that you have a team, the owners team, with a single member. An
+organization owner can then create new teams and add members to an existing team
+using Docker IDs and by selecting a team the user should be part of.
+
+> **Note:**
+>
+> In Docker Hub, users cannot belong directly to an organization.
```
Comment (shawnaxsom) [inline docker-hub/orgs.md:28]: As mentioned above, I would say this wording could be ambiguous. "Directly" could be taken in different ways.

I'd word it that they require membership to at least one team.

File: docker-hub/orgs.md [hunk 2]
```diff
@@ -9,151 +9,228 @@ redirect_from:
...
+They belong to teams within an organization. Organization owners (Admins) can
```
[the remark this answers is not in the document]
Comment (shawnaxsom) [reply docker-hub/orgs.md:29]: Take my comments with a grain of salt, it's a small section and this is a note within the same section where that terminology is mentioned at least, but good to consider 🙂

File: docker-hub/orgs.md [hunk 3]
```diff
@@ -9,151 +9,228 @@ redirect_from:
...
+[configure repository permissions](#configure-repository-permissions) on a per-team basis. For example, as an org owner,
+you can specify that all teams within an organization have Read and Write access
+to repositories A and B, whereas only specific teams have Admin access to those
+repositories. Org owners have full administrative access to all repositories
+within the organization.
+
+## Create an organization
```
Comment (shawnaxsom) [inline docker-hub/orgs.md:36]: Keep in mind there are multiple ways to create orgs (e.g. convert your user account profile to an org).

Someone reading just this section might want to know there are other ways (at least that way).

I wouldn't deep dive here, but say maybe one sentence that orgs can be created brand new or converted from user accounts (and link to a different section on how to convert).

File: docker-hub/orgs.md [hunk 4]
```diff
@@ -9,151 +9,228 @@ redirect_from:
...
+You've now created an organization. Select the newly created organization from
+the Organizations page. You'll now see that you have a team, the **owners** team
+with a single member (you).
+
+### View an organization
+
+To view an organization:
+
+1. Log into Docker Hub with a user account that is a member of any team in the
+   organization. You must be part of the **owners** team to access the
```
Comment (shawnaxsom) [inline docker-hub/orgs.md:69]: I mentioned capitalizing or quotes, but **owners** emphasis can work too (might want to apply that other places "owners" is mentioned.

Comment (shawnaxsom) [reply docker-hub/orgs.md:69]: "to access" -- I'd consider mentioning "to see and access", it won't be visible as a non owner if they're wondering where it is

File: docker-hub/orgs.md [hunk 5]
```diff
@@ -9,151 +9,228 @@ redirect_from:
...
+
+      > **Note:**
+      >
+      > You can't _directly_ log into an organization. This is especially
+      > important to note if you create an organization by converting a user
+      > account, as conversion means you lose the ability to log into that
+      > "account", since it no longer exists. If you don't see the organization,
+      > then you are neither a member or an owner of it. An organization
+      > administrator will need to add you as a member of the organization team.
+2. Click **Organizations** in the top navigation bar, then choose your
```
Comment (shawnaxsom) [inline docker-hub/orgs.md:80]: New line needed here?

File: docker-hub/orgs.md [hunk 6]
```diff
@@ -9,151 +9,228 @@ redirect_from:
 Docker Hub organizations let you create teams so you can give your team access
 to shared image repositories.
 
-- **Organizations** are collections of teams and repositories that can be managed together.
-- **Teams** are groups of Docker Hub users that belong to an organization.
+An **Organization** is a collection of teams and repositories
+that can be managed together. Docker users cannot belong directly to an
```
[the remark this answers is not in the document]
Comment (usha-mandya) [reply docker-hub/orgs.md:13]: Thanks. Updating it to "Docker users become members of an organization when they are assigned to at least one team in the organization." Hope that clarifies

File: docker-hub/orgs.md [hunk 7]
```diff
@@ -9,151 +9,228 @@ redirect_from:
...
+organization. They belong only to teams within an organization. When you first
+create an organization, you’ll see that you have a team, the owners team, with a
+single member. An organization owner can then create new teams and add members
+to an existing team using Docker IDs or email address and by selecting a team
+the user should be part of.
```
[the remark this answers is not in the document]
Comment (usha-mandya) [reply docker-hub/orgs.md:18]: Good point. Included this in the "Add members to the team" section

File: docker-hub/orgs.md (hunk 1 above)
Comment (usha-mandya) [reply docker-hub/orgs.md:28]: I believe this note is now redundant as the Org section (which is right above this) has the same info. I'll remove this note.

File: docker-hub/orgs.md (hunk 3 above)
Comment (usha-mandya) [reply docker-hub/orgs.md:36]: Good point. I've added an intro with info on two ways of creating an org and added a link to the Convert a user account into a org section

**Comment by usha-mandya** [comment]
@MatthewHNH I've included info on the Invitees tab and a call out about seats when a user is on Team and Business plan. PTAL.

**Review by jerae-duffin — APPROVED** [review]
LGTM

**Comment by MatthewHNH** [comment]
> 



> @MatthewHNH I've included info on the Invitees tab and a call out about seats when a user is on Team and Business plan. PTAL.

LGTM

**Pull request merged**